.circleci/config.yml

```diff
@@ -34,7 +34,19 @@ jobs:
       - deploy:
           name: Deploy
           command: |
+            # Do not release on Broad Holidays
+            CURRDATE=$(date +%m/%d/%Y)
+             array=("03/12/2021" "4/19/2021" "5/28/2021" "5/31/2021" "7/2/2021" "7/5/2021"
```

Comment:
Can this be named more specifically than just `array`?

Reply:
Yes - thanks!